pubspec.yaml

```diff
@@ -1,36 +1,37 @@
-# 
+#
 # Copyright 2023 The TensorFlow Authors. All Rights Reserved.
-# 
+#
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
-# 
+#
 #             http://www.apache.org/licenses/LICENSE-2.0
-# 
+#
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# 
+#
 
 name: tflite_flutter
 description: TensorFlow Lite Flutter plugin provides an easy, flexible, and fast Dart API to integrate TFLite models in flutter apps across mobile and desktop platforms.
 version: 0.9.5
 homepage: https://github.com/tensorflow/flutter-tflite
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=1.26.0-17.6.pre"
+  sdk: '>=3.0.0 <4.0.0'
```

Comment:
I'm really looking forward to merging this request. Good job 👍

One question. Do we really have to raise the minimum sdk version? I think a lot of big projects haven't made it to dart 3 yet.

Reply:
ffi_gen is generating `final class` which is only compatible with Dart 3. But we can try an older version of ffi_gen and see how it goes.

Reply:
It's ready. 

sdk: '>=2.17.0 <4.0.0'